internal/photoprism/mediafile.go

```diff
@@ -314,6 +320,113 @@ func (m *MediaFile) EditedName() string {
 	return ""
 }
 
+func (m *MediaFile) VideoFileName(jsonName string) (string, error) {
```

Comment:
Maybe rename VideoFileName to EmbeddedVideoName or ExtractEmbeddedVideo so it's clear it doesn't return the filename of regular sidecar video file?